cogs/lounge.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, bot):
 
         Anything else isn't supported. The C++ compiler uses g++ -std=c++14.
 
-        The python support is only python2.7 (unfortunately).
+        The python support is now 3.5.2.
```

Comment:
remove the word "now", the doc strings should be written as if the user never knew that coliru didn't have python2 support.